gateleen-routing/src/main/java/org/swisspush/gateleen/routing/RouterConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.swisspush.gateleen.routing;
+
+import org.swisspush.gateleen.routing.auth.OAuthConfiguration;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Container holding configuration values for the {@link Router}
+ *
+ * @author https://github.com/mcweba [Marc-Andre Weber]
+ */
+public class RouterConfiguration {
+
+    private final Integer requestHopsLimit;
+    private final Map<String, OAuthConfiguration> oAuthConfigurations;
```

Comment:
Maybe indicate what the key is

Reply:
Going to introduce a dedicated class for this id called _OAuthId_ instead of just a _String_